content/ai/concepts/neural-networks/terms/artificial-neural-networks/artificial-neural-networks.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Artificial Neural Networks'
+Description: 'A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain.'
+Subjects:
+  - 'Machine Learning'
+  - 'Brain Activity'
+  - 'Machine Learning'
+  - 'Deep Learning'
+  - 'Perceptrons'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'ANN'
+  - 'Neural Networks'
+  - 'Weight & Bias'
+  - 'Neurons'
```

Comment:
```suggestion
```
This tag is duplicative of Neural Networks, and is less precise.

Reply:
done